cwms-data-api/src/test/resources/cwms/cda/data/dto/basin/basin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "basin-id": {
+    "office-id": "TEST_OFFICE2",
```

Comment:
Try to use a valid office in these examples. Everyone has their favorite, avoid "CWMS" but otherwise you can pick one from the /offices endpoint.

Reply:
Fixed in latest push

Reply:
@MikeNeilson